SuiteCRM/SuiteCRM
# Fix #9440 - Forcing default null value for numeric core fields.

Closes #9440 

## Description
As described in the issue, non-custom/core non-required numeric fields aren't saving properly a default empty value in newly created modules.

## Motivation and Context
(int) and (float) summoning always return 0 if the value is empty. Therefore we add a condition to return "null" if it's empty.

## How To Test This
1. Create a new module in Module Builder
2. Add an Integer or Decimal field
3. Add the field to Edit and Detail View
4. Deploy module
5. Create a new record to the module, filling only the required name
6. Check that the numeric field has an empty value

## Types of changes

### Final checklist
